src/main/webapp/resources/js/pages/password/ForgotPassword.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Button, Form, Input } from "antd";
+import { LoadingOutlined, MailOutlined } from "@ant-design/icons";
+import { InfoAlert } from "../../components/alerts";
+import { SPACE_MD } from "../../styles/spacing";
+import { blue6 } from "../../styles/colors";
+import { useCreatePasswordResetEmailMutation } from "../../apis/password-reset";
+
+const { Item } = Form;
+
+/**
+ * React component to render the forgot password form
+ * @param {function} updateDisplayLoginPage Function to update whether to display login page
+ * @returns {*}
+ * @constructor
+ */
+export function ForgotPassword({ updateDisplayLoginPage }) {
+  const [forgotPassword] = useCreatePasswordResetEmailMutation();
+  const [forgotPasswordForm] = Form.useForm();
+  const [loading, setLoading] = React.useState(false);
+  const [message, setMessage] = React.useState(null);
+  const usernameOrEmailRef = React.useRef();
+
+  /**
+   * When the component gets added to the page,
+   * focus on the usernameOrEmail input.
+   */
+  React.useEffect(() => {
+    usernameOrEmailRef.current.focus();
+    usernameOrEmailRef.current.select();
+  }, []);
+
+  const submitForgotPasswordForm = () => {
+    setLoading(true);
+    setMessage(null);
+
+    forgotPassword({
+      usernameOrEmail: forgotPasswordForm.getFieldValue("usernameOrEmail"),
+    }).then((response) => {
+      if (response.error) {
+        setMessage(response.error.data.error);
+      } else {
+        setMessage(response.data.message);
+      }
+      forgotPasswordForm.resetFields();
+      setLoading(false);
+    });
+  };
+
+  return (
+    <div>
```

Comment:
Probably don't need the surrounding div, just use an empty fragment `<>`

Reply:
Removed in eb461f960fdd78a45922a9339cbc750424d85595